### DISCUSSION TITLE
kong support automatic master-slave switch

### DISCUSSION THREAD

**Discussion by liukaiyuliukaiyu** [Kong/kong#9527] [Ideas and feature requests]
Whether kong can support automatic master-slave switching? when the primary postgresql database fails，the primary/secondary switchover is detected through the database protocol, and the underlying layer can be adjusted to connect to the new primary database.

**Comment by hbagdi** [comment]
I don't think this is the case today. This is actively being pursued by our new initiative https://github.com/kong/koko.
cc @mikefero
